### DISCUSSION TITLE
退出时记忆播放器的状态, 为何要限制他

### DISCUSSION THREAD

**Discussion by letterk** [hooke007/mpv_PlayKit#45] [Q&A 问答] [answered]
音画不同步, 字幕不同步之类的问题, 调好了记住不是更好?

**Comment by hooke007** [comment, marked as answer]
https://github.com/hooke007/mpv_PlayKit/wiki/0_FAQ#%E4%BF%9D%E5%AD%98%E8%BF%90%E8%A1%8C%E6%97%B6%E4%BF%AE%E6%94%B9%E8%BF%87%E7%9A%84%E9%80%89%E9%A1%B9

防止手滑
防止滤镜混叠
总而言之就是防止意外，白名单可以自行适配个人的使用需求

**Reply by letterk** [reply]
或者有没有, 清除当前媒体文件记忆的方法
绑个快捷键?

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#command-interface-delete-watch-later-config
